README.md

```diff
@@ -0,0 +1,258 @@
+# ServerAdvertisement3
+Very useful plugin for your server with which you can print messages to players. Messages are stored in file.
+
+## Features
+- Message Types:
+	- Chat messages ***(Type T)*** - Color Support;
+	- HUD messages ***(Type H)*** - R G B Color;
+	- Center Text messages ***(Type C)*** - Support Only HTML **`hex`** colors;
+- Multi-language support; ***(Use [Alpha-2 Codes](https://en.wikipedia.org/wiki/ISO_3166-1))***
+- Up to 9 lines per message; **(`Use \n` to break lines)**
```

Comment:
Per #16 this is not fixed anymore. Lines amount can vary and are implicitly limited by the enough large buffer (i.e. max. text size).